test/unit/dialects/oracledb.js

```diff
@@ -111,6 +111,19 @@ describe('OracleDb parameters', function() {
         });
     });
 
+    it('on blob', function() {
+      return knexClient
```

Comment:
can you use async/await instead?

Reply:
This is the style of this entire file.  Is there a push to get all the files to use async away?  And also, are tests meant to be used on older versions?

Reply:
we only support Node 8+, and will convert code eventually. new one is preferred in newer syntax.